### PR TITLE
fix: Zero pad months in table names

### LINKED ISSUE #653: Zero Pad table names (Blocked until Nov.)
Months specified in table names created in `get_rotating_message_table()` are not zero padded, leading to errors when sorting. 

So that we don't prematurely lose data, this error can only be addressed in Nov or Dec.

### PR DESCRIPTION
**Pull request by jrconlin**
closes #653

### CODE REVIEW

File: autopush/db.py [hunk 1]
```diff
@@ -125,7 +125,7 @@ def make_rotating_tablename(prefix, delta=0, date=None):
     month delta."""
     if not date:
         date = get_month(delta=delta)
-    return "{}_{}_{}".format(prefix, date.year, date.month)
+    return "{}_{:04d}_{:02d}".format(prefix, date.year, date.month)
```
Comment (bbangert) [inline autopush/db.py]: The year?

Comment (jrconlin) [inline autopush/db.py]: Yeah, the year.

Sorry, but I still bear scars from Y2K.

Comment (bbangert) [inline autopush/db.py]: Shouldn't we use :05d then? ;)

Comment (pjenvey) [inline autopush/db.py]: does using strftime make you more comfortable? =]

r+

Comment (jrconlin) [inline autopush/db.py]: @bbangert Ooh, you're right. Remind me to fix it in 7,984 years. 

(I'll go back to slowly drawing red circles around Tuesday, 19 January 2038)

**Pull request merged**